cmdline.go

```diff
@@ -95,8 +101,13 @@ func ParseCmdLine(args ...string) CmdLine {
 		}
 
 		if s, err := strconv.ParseInt(item, 10, 64); !IDsExhausted && err == nil {
-			ids = append(ids, int(s))
-			continue
+			if int(s) <= MAX_TASKS_OPEN && string(lcItem[0]) != "0" {
```

Comment:
Added a little more error checking here. I ran into the scenario where my first 7-8 digits of the UUID were all numeric and they resolved to IDs rather than UUIDs.

I also added the condition that if the first digit of lcItem is 0, it cannot be parsed as an ID. I ran into the scenario where the first 6 dgits of my UUID were all numeric with a leading 0.

With MAX_TASKS_OPEN set to 10000, there is a ~0.08% chance of conflict between UUID and ID given the first 6 digits.
Edit: Although if the requirement that the UUID needs to be greater than 4 digits, the only potential conflict (assuming MAX_TASKS_OPEN = 10000) is the case that ID == 10000, or 1 in 15^5. There is still the case that you might run into issues while while trying to match other text